Sources/NIO/EventLoop.swift

```diff
@@ -482,13 +496,47 @@ extension EventLoop {
     /// the `EventLoopFuture` returned by `task`.
     ///
     /// - parameters:
-    ///     - task: The synchronous task to run. As everything that runs on the `EventLoop`, it must not block.
+    ///     - task: The asynchronous task to run. As with everything that runs on the `EventLoop`, it must not block.
     /// - returns: An `EventLoopFuture` identical to the `EventLooopFuture` returned from `task`.
     @inlinable
     public func flatSubmit<T>(_ task: @escaping () -> EventLoopFuture<T>) -> EventLoopFuture<T> {
         return self.submit(task).flatMap { $0 }
     }
 
+    /// Schedule a `task` that is executed by this `EventLoop` at the given time.
+    ///
+    /// - parameters:
+    ///     - task: The asynchronous task to run. As with everything that runs on the `EventLoop`, it must not block.
+    /// - returns: A `Scheduled` object which may be used to cancel the task if it has not yet run, or to wait
+    ///            on the full execution of the task, including its returned `EventLoopFuture`.
+    ///
+    /// - note: You can only cancel a task before it has started executing.
+    @discardableResult
+    public func flatScheduleTask<T>(deadline: NIODeadline, _ task: @escaping () throws -> EventLoopFuture<T>) -> Scheduled<T> {
```

Comment:
Can we make `file` and `line` parameters of this function and then pass them to the line below? That will be much more useful, debugging-wise.

Reply:
Done,